### ISSUE TITLE
NuGet pack produces invalid nupkg file when using leading slash in file target attribute

### ISSUE THREAD

**Issue by oviradoi** [NuGet/Home#2749]
I noticed that the `nuget pack` command is producing an invalid nupkg file when you are adding a leading slash or backslash to the target attribute of a file element.

This can be reproduced by using this minimal nuspec file:
(you also need a "test.txt" file that can be packed)

```
<?xml version="1.0" encoding="utf-8"?>
<package>
  <metadata>
    <id>test</id>
    <version>1.0</version>
    <authors>oviradoi</authors>
    <description>test</description>
  </metadata>
  <files>
    <file src="test.txt" target="/content/test.txt" />
  </files>
</package>
```

The problem I see is in the contents of the "target" attribute. If it contains a leading slash or backslash, when you try to create the nuget package, the .nupkg file will be invalid. NuGet will report that the package was created successfully, and Visual Studio will install it without complaining, but all the files inside the "content" directory will be ignored. This can happen for any folder, not just "content".

If you try to open the nuget file with the NuGet Package Explorer, the file list will be empty.
![pkgexpl](https://cloud.githubusercontent.com/assets/8641811/15124512/107d6042-1631-11e6-9e8f-5a7e15bee988.jpg)

If you rename the file to "zip" and open it with WinRAR, you can see that instead of "content", the folder is called "\content" (there's a leading backslash in there). I suspect that's the problem.
![winrar](https://cloud.githubusercontent.com/assets/8641811/15124511/107674da-1631-11e6-822a-465882e4ca4e.jpg)

If in the nuspec file you write "content/test.txt", everything works fine and the problem does not appear.

This problem appears when using nuget v3.4.3, as well as the latest nightly (which is v3.5.0.1259 right now). If you use the older nuget v3.3.0, this problem does not appear.

When you are writing the nuspec file by hand this is not really an issue because you can be careful not to put the leading slash or backslash in there. But I stumbled across this problem when using the CoApp tools to build a native NuGet package. The CoApp tools generate the nuspec file, and it puts leading slashes and backslashes on target attributes of the file elements. With this bug, building native packages using the CoApp tools is broken unless you use the older NuGet v3.3.0.

**Comment by rrelyea** [comment]
Consider fix for 3.4.4 if slam dunk fix.

**Comment by zhili1208** [comment]
verified in 3.4.4
